app/src/main/res/values-nl/strings.xml

```diff
@@ -143,9 +143,9 @@
 
         De galerij kan ook worden gebruikt voor het bekijken van afbeeldingen of video\'s vanuit andere apps, om bijlagen toe te voegen in e-mail, etc. Perfect voor dagelijks gebruik.
 
-        De permissie voor het uitlezen van vingerafdrukken is benodigd voor het vergendelen van verborgen items of de gehele app.
+        Het toegangsrecht voor het uitlezen van vingerafdrukken is benodigd voor het vergendelen van verborgen items of de gehele app.
 
-        Bevat geen advertenties of onnodige permissies. Volledig open-source. Kleuren van de app kunnen worden aangepast.
+        Bevat geen advertenties of onnodige toegangsrechten. Volledig open-bron. Kleuren van de app kunnen worden aangepast.
```

Comment:
Either "permissie" or "machtiging" (like in Play Store)
No-one calls open-source "open-bron".

Reply:
"permissie" is a Flemish word.

And yes, openbronsoftware is actually used. Google it.

Reply:
Haha so, basically you've assumed that if it's not on http://anw.inl.nl it's a Flemish word?

I just meant "open-bron" is almost never used, and it's meaning will probably be less straightforward to most than the English term.

Reply:
No. I have Flemish colleagues. I just gave you an example of a national institution who actually prefers that term. But a quick internet search for "openbronsoftware" reveals that a lot of sites use it, even newspapers like NRC.